spec/index.html

```diff
@@ -253,25 +253,43 @@
     <section>
       <h2>Introduction</h2>
       <p>
-        JSON-LD is a JSON-based format to serialize Linked Data.
-        In recent years, YAML emerged as a more concise format
-        to represent information serialized as JSON,
+        [[JSON-LD]] is a JSON-based format to serialize Linked Data.
+        In recent years, [[YAML]] emerged as a more concise format
+        to represent information serialized as [[JSON]],
         including API specification, data schemas, and Linked Data.
       </p>
       <p>
         Since YAML is more expressive than JSON,
         both in the available data types and in the document structure
         (see [[I-D.ietf-httpapi-yaml-mediatypes]]),
         this document identifies constraints on YAML
-        such that it can be used to represent JSON-LD documents.
+        such that any <a>YAML-LD document</a> can be represented in JSON-LD.
```

Comment:
This seems a non-sequitur: you discuss constraints on YAML that make YAML-LD can be represented as JSON-LD without first defining how YAML-LD relates to YAML. Maybe it needs to have a definition moved forward or referenced.

Reply:
Added an explanatory paragraph, please let me know what you think.